### PR TITLE
Add Tizen Studio 2.4 Release Notes

### PR DESCRIPTION
**Pull request by karthikthecool**
Signed-off-by: KARTHIK VENKATESH BHAT/Tizen/Staff Engineer/삼성전자 <kv.bhat@corp.samsungelectronics.net>

### CODE REVIEW

**Comment by mijin85cho** [comment]
Hello, karthikthecool
Would you update docs/application/tizen-studio/TOC.md file, too?
I hope https://github.com/Samsung/tizen-docs/commit/5c936d67eba1507dc2a2eae7b4bef0de92e098ed can be a reference for you.

**Comment by karthikthecool** [comment]
Hi @mijin85cho 
Updated the ToC as per review comments.
Thanks

**Comment by mijin85cho** [comment]
Add surbhig12, too, as it is said Annie is absent this week.

**Comment by surbhig12** [comment]
@mijin85cho 

Thank you. This file will be reviewed by Monday.

File: docs/application/tizen-studio/release-notes/2-4-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	-   Updated ramls for smart things resources
+-   Package Manager
+	-   Remove *.po files getting packaged into tpk
+
+### Changed Features
+-	RT-IDE
+	-   Removed local template for creating project. Project can be created using latest RT code from github.
+
+### Fixed Bugs
+-   Device Manager
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/2-4-release-notes.md]: Device Manager 
•	Fixed the crash that occurred while filtering devices in JAVA 9.
•	Increased the clickable area for Add new tab button to cover the entire button area.
•	Added support for empty filter category in logs.
•	Fixed permit to install the failure for secure TV profile.

File: docs/application/tizen-studio/release-notes/2-4-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+-   Device Manager
+	-   Enable Device Manager Context UI to be extended by plugins
+-   RT-IDE
+	-   RT-IDE GCC Toolchain update to 6.3.1
+-   SmartThings
+	-   Make setupId mandatory field and map it to device onboarding id
+	-   Updated ramls for smart things resources
+-   Package Manager
+	-   Remove *.po files getting packaged into tpk
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/2-4-release-notes.md]: Removed portable object (*.po) files , which are packaged into tpk.

File: docs/application/tizen-studio/release-notes/2-4-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+### Changed Features
+-	RT-IDE
+	-   Removed local template for creating project. Project can be created using latest RT code from github.
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/2-4-release-notes.md]: Removed the local template for creating a project. The project can be created using the latest RT code from GitHub.

File: docs/application/tizen-studio/release-notes/2-4-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+### Fixed Bugs
+-   Device Manager
+	-   Fix crash when filtering devices in JAVA 9
+	-   "Add new tab" button clickable area changed to full button
+	-   Added support for empty filter category in logs
+	-   Fixed Permit to install failure for secure TV profile
+-   Emulator
+	-   Corrected the marked location data in emulator control panel
+-   Package Manager
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/2-4-release-notes.md]: Package Manager

-     Added https support in Package Manager to download SDK from https website.
-     Fixed multiple instance of same Package Manager getting launched in Windows and macOS.

File: docs/application/tizen-studio/release-notes/2-4-release-notes.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+	-   Added https support in package manager to download SDK from https website.
+	-   Fix multiple instance of same package manager getting launched in Windows and macOS.
+-   Certificate Manager
+	-   Fixed the socket timeout error when creating IoT certificates
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/2-4-release-notes.md]: Add a period.

File: docs/application/tizen-studio/release-notes/2-4-release-notes.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+	-   On Windows&reg;, depending on your OS theme (such as Non-Aero themes and Windows XP themes), a display surface can be erased for a while if the emulator window is covered with another window. If you click the emulator window, the display surface runs correctly again.
+	-   On Windows&reg;, if an error with message "failed to allocate memory" occurs while executing the emulator, try the following:
+	    -   Close some other programs and try to launch the emulator again.
+	    -   If the RAM size is set to 768 or 1024 MB for the VM in the Emulator Manager, change it to 512 MB.
+	    -   Increase the user area of the virtual memory in the system to 3 GB by entering the *bcdedit /setincreaseuserva 3072* command on the console with administrator rights (only in Windows&reg; 7), and reboot.
+	-   If you use a MacBook Pro which has both Intel HD and NVIDIA&reg; GPUs, the emulator can unexpectedly terminate when you execute the emulator with OpenGL ES version set v1.1 & v2.0. Verify the emulator configuration in the Emulator Manager and on the general tab in the emulator configuration window, set OpenGL ES version to v2.0 & v3.0.
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/2-4-release-notes.md]: If you use a MacBook Pro which has both Intel HD and NVIDIA&reg; GPUs, the emulator can unexpectedly terminate when you execute the emulator with OpenGL ES version set v1.1 and OpenGL ES version set v2.0. Verify the emulator configuration in the Emulator Manager and on the general tab in the emulator configuration window, set OpenGL ES version to version v2.0 and version v3.0.

File: docs/application/tizen-studio/release-notes/2-4-release-notes.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+	-   When you launch the Emulator Manager in the Tizen IDE, the Emulator Manager's shortcut image may not be displayed properly.
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/2-4-release-notes.md]: When you launch the Emulator Manager in the Tizen IDE, the shortcut image of Emulator Manager may not be displayed properly.

File: docs/application/tizen-studio/release-notes/2-4-release-notes.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+## IDE and Tools
+
+
+### New Features
+
+-   Device Manager
+	-   Enable Device Manager Context UI to be extended by plugins
+-   RT-IDE
+	-   RT-IDE GCC Toolchain update to 6.3.1
+-   SmartThings
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/2-4-release-notes.md]: SmartThings™

File: docs/application/tizen-studio/release-notes/2-4-release-notes.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+	    -   If the host machine is using NVIDIA&reg; Optimus&reg; technology on either Ubuntu or Windows&reg;, you must set the Tizen emulator to run with your NVIDIA graphics card. For Ubuntu, verify the [bumblebee project](https://wiki.ubuntu.com/Bumblebee ). For Windows&reg;, select *High Speed NVIDIA Processor* as *Preferred Graphics processor* in the NVIDIA control panel.
+	    -   On Ubuntu, if the graphics driver is out-of-date, your Ubuntu desktop session occasionally logs out while launching the Emulator Manager, or the emulator skin is displayed improperly. Verify the prerequisites and upgrade to the latest graphics driver.
+	-   On Ubuntu 14.04, a shortcut menu can sometimes appear transparent.
+	-   On Windows&reg;, depending on your OS theme (such as Non-Aero themes and Windows XP themes), a display surface can be erased for a while if the emulator window is covered with another window. If you click the emulator window, the display surface runs correctly again.
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/2-4-release-notes.md]: Is it Non-Aero themes or Non-aero themes?

**Comment by Reni-90** [comment]
The edited .md file is attached for your reference. Please verify that the technical meaning is not changed in the rephrases.
[2-4-release-notes.zip](https://github.com/Samsung/tizen-docs/files/2043970/2-4-release-notes.zip)

**Comment by mijin85cho** [comment]
Hello Reni-90
I can see the update made by you. Thanks for that.

Hello karthikthecool
Would you please check the update by Reni?
If you confirm the update, I'll merge it,

Regards

**Comment by karthikthecool** [comment]
@Reni-90  Thanks for the changes.
@mijin85cho  Looks good to me. Thanks!

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**